backend/src/utils/resourceUtils.ts

```diff
@@ -56,24 +57,28 @@ let buildsWatcher: ResourceWatcher<BuildStatus>;
 let consoleLinksWatcher: ResourceWatcher<ConsoleLinkKind>;
 let quickStartWatcher: ResourceWatcher<QuickStart>;
 
+const DASHBOARD_CONFIG = {
+  group: 'opendatahub.io',
+  version: 'v1alpha',
+  plural: 'odhdashboardconfigs',
+  dashboardName: process.env['DASHBOARD_CONFIG'] || dashboardConfigMapName,
+};
+
 const fetchDashboardCR = async (fastify: KubeFastifyInstance): Promise<DashboardConfig[]> => {
-  const dashboardName = process.env['DASHBOARD_CONFIG'] || dashboardConfigMapName;
-  return fetchOrCreateDashboardCR(fastify, dashboardName).then((dashboardCR) => {
-    return migrateTemplateDisablement(fastify, dashboardCR).then((dashboardCR) => [dashboardCR]);
-  });
+  return fetchOrCreateDashboardCR(fastify)
+    .then((dashboardCR) => migrateTemplateDisablement(fastify, dashboardCR))
```

Comment:
`migrateTemplateDisablement` was created almost a year ago for a upgrade path from 2.11, it was intended to be removed once we dropped support from that version, isn't that the case right now?

Reply:
🤷 wanna log something @lucferbux ? This is well outside of scope.

Reply:
Yes, I agree, I've created https://issues.redhat.com/browse/RHOAIENG-8499 for this.